changelogs/current.yaml

```diff
@@ -89,6 +89,9 @@ minor_behavior_changes:
 
 bug_fixes:
 # *Changes expected to improve the state of the world and are unlikely to have negative effects*
+- area: lrs
+  change: |
+    Fixes log spamming for LoadStatsReporting graceful stream close when using default log level.
```

Comment:
Thanks!
I actually thought that the thing that should be clarified is that the `errors` counter isn't incremented in the graceful case.

Reply:
Ah, true.